src/Exception/ApiException.php

```diff
@@ -6,15 +6,12 @@
 
 class ApiException extends GovUkAccountException
 {
-    public array $data;
-
     public function __construct(
         string    $message = '',
         int       $code = 0,
         Throwable $previous = null,
-        array     $data = []
+        public array     $data = []
     ) {
-        $this->data = $data;
         parent::__construct($message, $code, $previous);
     }
```

Comment:
So `$data` wasn't set in the construct before, and still isn't?
Can't it be removed?

Reply:
It was previously being set in the constructor. Moving to constructor property promotion pattern.